x-pack/plugin/searchable-snapshots/qa/rest/src/test/resources/rest-api-spec/test/stats.yml

```diff
@@ -0,0 +1,158 @@
+---
+setup:
+
+  - do:
+      indices.create:
+        index: index
+        body:
+          settings:
+            number_of_shards:   1
+
+  - do:
+      bulk:
+        body:
+          - index:
+              _index: index
+          - field: foo
+          - index:
+              _index: index
+          - field: bar
+          - index:
+              _index: index
+          - field: baz
+
+  - do:
+      snapshot.create_repository:
+        repository: repository-fs
+        body:
+          type: fs
+          settings:
+            location: "repository-fs"
+
+  - do:
+      snapshot.create:
+        repository: repository-fs
+        snapshot: snapshot
+        wait_for_completion: true
+
+  - do:
+      snapshot.delete_repository:
+        repository: repository-fs
+
+  - do:
+      indices.delete:
+        index: index
+
+---
+"Tests searchable snapshots stats":
+  - skip:
+      version: " - 7.99.99"
+      reason:  searchable snapshots introduced in 8.0
+
+  - do:
+      searchable_snapshots.stats: {}
+
+  - length: { indices: 0 }
+
+  - do:
+      catch:   missing
+      searchable_snapshots.stats:
+        index: "unkown"
+
+  - do:
+      snapshot.create_repository:
+        repository: repository-searchable-snapshots
+        body:
+          type: searchable
+          settings:
+            delegate_type: fs
+            location: "repository-fs"
+
+  - match: { acknowledged: true }
+
+  - do:
+      snapshot.restore:
+        repository: repository-searchable-snapshots
+        snapshot: snapshot
+        wait_for_completion: true
+
+  - match: { snapshot.snapshot: snapshot }
+  - match: { snapshot.shards.failed: 0 }
+  - match: { snapshot.shards.successful: 1 }
+
+  - do:
+      search:
+        rest_total_hits_as_int: true
+        index: index
+        body:
+          query:
+            match_all: {}
+
+  - match: { hits.total: 3 }
+
+  - do:
+      nodes.info: {}
+  - set:
+      nodes._arbitrary_key_: node_id
+
+  - do:
+      searchable_snapshots.stats:
+        index: "ind*"
+
+  - length:  { indices: 1 }
+  - length:  { indices.index.shards: 1 }
+  - length:  { indices.index.shards.0: 1 }
```

Comment:
I expect this index has replicas; they won't be assigned if the test cluster only has a single data node, but I think this would fail if there were more nodes involved (we don't specify a single node cluster anywhere AFAIK). We should set the number of replicas to make sure this works even in larger clusters.

Reply:
Good catch - I set the number of replicas to 0 and also renamed the index `index` to `docs` to avoid confusion (f2ea47f228c27bead4933eb04cc18ae82ca39601).